### PR TITLE
Pd 3342 analytics closing timeline chart sends ajax

### PR DESCRIPTION
**Pull request by aljordan82**
## What's this PR do?
Implement shouldCOmponentUpdate
Update CSS

### CODE REVIEW

File: gulp/src/sass/analytics.scss [hunk 1]
```diff
@@ -40,6 +40,15 @@ footer {
   display: none;
 }
 
+/* Bootstrap overwrites*/
```
Comment (mwarnsleyDE) [inline gulp/src/sass/analytics.scss]: @aljordan82 we should find an alternative to overwriting bootstrap. Overwriting them could cause some problems in the future if we needed to use them somewhere else in the files.

File: gulp/src/analytics/components/Charts/ChartContainer.js [hunk 2]
```diff
@@ -13,66 +13,122 @@ import {doGetSelectedFilterData} from '../../actions/table-filter-actions';
...
+          'day': removeLeadingZero,
+        };
+      }
+      return {
+        'Job Views': month['Job Views'],
+        'month': removeLeadingZero,
+      };
+    });
+  }
   constructor(props, context) {
```
Comment (mwarnsleyDE) [inline gulp/src/analytics/components/Charts/ChartContainer.js]: @aljordan82 we should move this to the top of the class just to keep the structure the same as the rest of the components.

Comment (aljordan82) [inline gulp/src/analytics/components/Charts/ChartContainer.js]: @mwarnsleyDE , the linter marked it as an error, static vars/methods needs to be on top.

Comment (mwarnsleyDE) [inline gulp/src/analytics/components/Charts/ChartContainer.js]: @aljordan82 okay gotcha makes sense then we can leave this one as is

**Pull request merged**